build-tools/create-packs/Microsoft.Android.Sdk.proj

```diff
@@ -122,10 +122,12 @@ WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and
     <XamarinAndroidVersion>$(AndroidPackVersionLong)</XamarinAndroidVersion>
   </PropertyGroup>
   <PropertyGroup Condition=" '%24(TargetPlatformVersion)' == '31.0' ">
-    <_AndroidRuntimePackId>31</_AndroidRuntimePackId>
-    <_AndroidRuntimePackVersion>31.0.101-preview.11.117</_AndroidRuntimePackVersion>
+    <_AndroidTargetingPackId>31</_AndroidTargetingPackId>
+    <_AndroidTargetingPackVersion>31.0.101-preview.11.117</_AndroidTargetingPackVersion>
   </PropertyGroup>
   <PropertyGroup>
+    <_AndroidTargetingPackId Condition=" '%24(_AndroidTargetingPackId)' == '' ">$(AndroidLatestStableApiLevel)</_AndroidTargetingPackId>
+    <_AndroidTargetingPackVersion Condition=" '%24(_AndroidTargetingPackVersion)' == '' ">**FromWorkload**</_AndroidTargetingPackVersion>
     <_AndroidRuntimePackId Condition=" '%24(_AndroidRuntimePackId)' == '' ">$(AndroidLatestStableApiLevel)</_AndroidRuntimePackId>
     <_AndroidRuntimePackVersion Condition=" '%24(_AndroidRuntimePackVersion)' == '' ">**FromWorkload**</_AndroidRuntimePackVersion>
```

Comment:
Do we still need to use these `_AndroidRuntimePackId` `_AndroidRuntimePackVersion` based conditions if we aren't setting them anywhere anymore?

Reply:
I left `$(_AndroidRuntimePackId)` and `$(_AndroidRuntimePackVersion)` as private properties. They might be useful to be able to put them in a project file for debugging something down the road.